packages/docs/src/components/app/search/SearchDialog.vue

```diff
@@ -120,7 +120,7 @@
   const searchString = defineModel('search', { type: String, default: '' })
 
   // Algolia doesn't tokenize hyphens by default. Sanitize search string.
-  const searchTerm = computed(() => searchString.value.replace(/-/g, ' '))
+  const searchTerm = computed(() => searchString.value.replace(/-/g, ' ').trim())
```

Comment:
```suggestion
    const searchTerm = computed(() => searchString.value.replace(/^[^a-zA-Z0-9]/, ' ').trim())
```
replace only the first char if it's alpha numeric

Reply:
This still breaks advanced query syntax: https://www.algolia.com/doc/api-reference/api-parameters/advancedSyntax/

Reply:
hmm, so should we just filter out the empty search results instead? keep the search term as it was before 🤔